iiblib/iib_build_details_model.py

```diff
@@ -221,29 +221,31 @@ class AddModel(IIBBuildDetailsModel):
         "binary_image_resolved",
         "bundles",
         "bundle_mapping",
+        "deprecation_list",
```

Comment:
Could you please add this to the `Args` section above?

Reply:
done